tests/tools/oecert/enc/enc.cpp

```diff
@@ -112,6 +113,37 @@ oe_result_t get_tls_cert_signed_with_key(
     return result;
 }
 
+oe_result_t get_remote_report_with_custom_data(
+    uint8_t* custom_data,
+    size_t custom_data_size,
+    uint8_t** report_buffer,
+    size_t* report_buffer_size)
+{
+    oe_result_t result = OE_FAILURE;
+
+    uint8_t sha256[32];
+
+    init_mbedtls();
+
+    if (Sha256(custom_data, custom_data_size, sha256) != 0)
```

Comment:
Hash of custom_data is embedded in the report but the custom_data itself is not carried over. What is the user scenario? Cannot the custom_data_file itself be the hash value?

Reply:
I wanted to generate a report with the sha256 of the custom data.  The custom data plus the report is then sent in the body for request to MAA.  I just needed a tool to do this for me.

Reply:
MAA takes evidence type, evidence, and enclave held data. In your case, evidence type is oe_report and enclave help data is the hash of a public key. So your custom_data is the public key, right?